sync/src/main/java/tech/pegasys/artemis/sync/PendingPool.java

```diff
@@ -46,15 +48,20 @@
   private static final UnsignedLong GENESIS_SLOT = UnsignedLong.valueOf(Constants.GENESIS_SLOT);
 
   private final EventBus eventBus;
+  private final Subscribers<RequiredBlockRootSubscriber> requiredBlockRootSubscribers =
+      Subscribers.create(true);
```

Comment:
Any reason we want to suppress the callback exceptions?

Reply:
It means that if any subscriber ends up throwing an exception, it doesn't bubble up to this class and interrupt its processing.  Otherwise - we have to assume that any time we dispatch events the subscribers might throw errors and we'd have to handle that here.  Since we have no context, at best we'd just end up logging the exception.  Instead, the `Subscribers` class just handles that for us by logging the errors.